src/node/node_state.h

```diff
@@ -288,6 +289,28 @@ namespace ccf
       if (code_id.has_value())
       {
         node_code_id = code_id.value();
+
+        if (!config.attestation.environment.uvm_endorsements.has_value())
```

Comment:
Is this so we can set a config value to check against? That doesn't provide a security property.

Reply:
Yes a malicious operator could change the value of the environment variable, e.g. passing a COSE Sign1 document that endorses a compromised measurement of the UVM on which the CCF node is deployed. The corresponding DID will be recorded in the ledger though (still to do), and as for MRENCLAVE for SGX deployments, it is up to the members to check that the DID is trusted when the service is open. Happy to discuss the details further.